### DISCUSSION TITLE
build from source

### DISCUSSION THREAD

**Discussion by zero2076** [BuddiesOfBudgie/budgie-desktop#162] [Q&A] [answered]
I've finished  building my LFS,but can I install budgie on it by building from source?

**Comment by serebit** [comment, marked as answer]
We don't have specific instructions for building Budgie on LFS, but we do have a page on our wiki for general build instructions [here](https://github.com/BuddiesOfBudgie/budgie-desktop/wiki/Building-Budgie-Desktop).

**Reply by zero2076** [reply]
ok,thanks a lot!
